examples/plot_metrics_with_error.py

```diff
@@ -10,9 +10,15 @@
 # %%
 # Load and preprocess the data set
 # --------------------------------
-# We download the data set using `fetch_adult` function in `fairlearn.datasets`.
+# We download the data set using `fetch_openml` function in `sklearn.datasets`.
+# The original Adult data set can be found at https://archive.ics.uci.edu/ml/datasets/Adult
 # We start by importing the various modules we're going to use:
 #
+import os
```

Comment:
sorry this was a mistake and wasn't supposed to be committed -- i found i need to do this to run the notebook using my local fairlearn module files (that has the new edits) instead of the released package